test/getter_setter.jl

```diff
@@ -119,6 +119,14 @@ fmiGetBoolean!(fmuStruct, booleanValueReferences, cacheBoolean)
 fmiGetString!(fmuStruct, stringValueReferences, cacheString)
 @test cacheString == rndString
 
+if envFMUSTRUCT == "FMUCOMPONENT"
+    dirs = fmi2GetRealOutputDerivatives(fmuStruct, realValueReferences, ones(2))
+    @test dirs isa AbstractArray{<:Real}
+    @test dirs |> length == 2
+
+    st = fmi2SetRealInputDerivatives(fmuStruct, realValueReferences, ones(2), zeros(2))
+    @test_broken st == FMI.fmi2OK
```

Comment:
@ThummeTo I don't think even the dymola test fmus support input/output derivatives. `fmi2GetRealOutputDerivatives` returns zeros and `fmi2SetRealInputDerivatives` returns `fmi2Discarded`.

Reply:
Dymola supports this, but there is no FMU that needs I/O-derivatives (no keywords in the model description). I can provide a simple FMU, that needs I/O-derivatives for automated testing. I opened the branch "i-o-derivatives", so we can merge there? (just change your PR from "main" to "i-o-derivatives").

Reply:
Just did that.